misk/src/main/kotlin/misk/environment/Environment.kt

```diff
@@ -4,39 +4,55 @@ import com.google.common.base.Preconditions
 import misk.logging.getLogger
 
 /** The environment in which the application is running */
+@Deprecated("use Deployment instead")
 enum class Environment {
   TESTING,
   DEVELOPMENT,
   STAGING,
   PRODUCTION;
 
+  fun isFake(): Boolean = this == TESTING || this == DEVELOPMENT
+
+  fun isReal(): Boolean = !isFake()
+
   companion object {
     internal val logger = getLogger<Environment>()
     private const val ENV_ENVIRONMENT = "ENVIRONMENT"
-    private lateinit var env: Environment
+
+    private lateinit var rawEnv: String
```

Comment:
Rename to just `string` ?

Reply:
I don't follow. Rename the type or variable name?

Reply:
I was thinking the variable name. I don’t believe rawEnv should be a thing long term.

Reply:
It's not a good name, but fortunately this entire class (and its `rawEnv`) will be going away soon.